packages/design-system/CONTRIBUTING.md

```diff
@@ -1,53 +1,140 @@
+- Get peer reviews on the API of your component before moving on to the implementation.
+- Follow the basic structure of a component/layout folder in the repository and use the [provided template for doc pages](DOCTEMPLATE.md).
 ```
-├── Component.tsx
-├── Component.spec.tsx     # if you need e2e tests
-├── Component.stories.mdx
-├── Component.stories.tsx
-├── Component.style.tsx
-├── Component.test.tsx     # if you need unit tests
-├── index.ts
-└── variations
-    ├── Component.A.tsx
-    └── Component.B.tsx
+├── Component.tsx          # Component
+├── Component.spec.tsx     # Cypress tests
+├── Component.stories.mdx  # Documentation and call to `<Story>`
+├── Component.stories.tsx  # Actual stories written explicitely in TSX
+├── Component.module.scss  # Component styles (in BEM)
+├── index.ts               # Clean component export
+└── private                # Internal composition elements
+    ├── ComponentBrick.A.tsx
+    └── ComponentBrick.B.tsx
 ```
+- Use design tokens with a burning passion.
+- Explicitness > Implicitness, always and in all things.
+- Don't let consumers easily override the styles or behaviours of the component.
+- Let designers review your work (don't merge and release without a designer's OK).
+- Let developers outside your team review your work.
 
-## Documentation
+## Contributing in the design phase
 
-```markdown
-import { Meta, Story } from '@storybook/addon-docs/blocks';
-import { FigmaImage, FigmaLink, Use } from '../../../docs';
+### What's the design phase?
 
-<Meta title="Components/Title"/>
+No component arrives in the design system without being thoroughly discussed "as a pattern". How should it work? When should we use it? When should we not? How does it look like? How does it look like when it's disabled/hovered/active etc... That's the design phase.
 
-# Title
+We expect developers to bring as many questions as they can during that phase.
 
-Description 
+**Their goal is to provide designers with the necessary means to create a complete specification of the component.**
 
-## Zoning
+It's probably the most important part of the collaboration process. Ensure the specs are as thorough as possible. Don't assume you'll be the one coding the component (you may, but somebody else might as well).
 
-<FigmaImage
-    src=""
-    alt="zoning image"
-/>
+**Things you need to contribute during design:**
+
+- Feasibility feedback (browser vs Figma)
+- State requirements ("how does it look when hovered?" etc...)
+- Behaviour requirements ("how should I display in error in this case?" etc...)
+- Human linting ("This text is not using a design token, I can't code it!" etc...)
+- Token guidance. (ex: designers may expect an icon to be an image and would use background tokens where you would use text tokens.)
+- Documentation: help the design time write down and distribute the specs for the component / layout / thing they are designing. This may involve creating the necessary Storybook PR for them.
```

Comment:
```suggestion
- Documentation: help the design team write down and distribute the specs for the component / layout / thing they are designing. This may involve creating the necessary Storybook PR for them.
```